app-config.dashboard.yaml

```diff
@@ -21,7 +21,7 @@ kubernetes:
     - type: config
       clusters:
         - name: self
-          url: https://kubernetes
+          url: https://kubernetes.default.svc.cluster.local
```

Comment:
Do we have to specify this field?

If we want to align with the logic in client-go, that's here: https://github.com/kubernetes/client-go/blob/8092c71d36058837d096a519db456411e0cbde8e/rest/config.go#L511

Reply:
In my testing I found that this was needed, even when the serviceAccountToken was omitted. Per https://backstage.io/docs/features/kubernetes/configuration/#clustersserviceaccounttoken-optional, it states that:

> If a cluster has authProvider: serviceAccount and the serviceAccountToken field is omitted, Backstage will ignore the configured URL and certificate data, instead attempting to access the Kubernetes API via an in-cluster client as in [this example](https://github.com/kubernetes-client/javascript/blob/master/examples/in-cluster.js).

However I was getting endpoint not found errors when it was just `https://kubernetes`, and when I changed it to the fully qualified address it worked correctly.

Reply:
Can you add a comment then?